pyonfx/convert.py

```diff
@@ -82,76 +116,125 @@ def time(ass_ms: Union[int, str]) -> Union[str, int, ValueError]:
             raise ValueError("Milliseconds or ASS timestamp expected")
 
     @staticmethod
-    def ms_to_frames(ms: int, fps: Union[int, float, Fraction], is_start: bool) -> int:
-        """Converts from milliseconds to frames.
+    def ms_to_frames(
```

Comment:
I would change the order of the parameters to:
- ms
- time_type
- timestamps

Then, should we keep the "approximate"? Why would anyone want to not "approximate" if it is beyond the video end (which should happen mostly for the last line only)?

Reply:
It helps the user to know, "Oops, anything above this point won't be rendered anyway, so why would I lose my time to process X line."

Reply:
But then, if it is "True" by default I assume that most of the users will never touch this value, thus they will never get to know.

I am in favour of ease of use in cases like this, even if it to sacrifice a bit of utility: I would remove it and keep the approximation as the only behaviour. It seems to me that whoever will want, can discover and avoid to process (if needed) lines exceeding the end of the video.

Reply:
Done